### PR TITLE
Flatten blocks for hydration

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
We currently only list the elements from the first block to send to the client for hydration. Here we check all elments, from all blocks

## Why?
Better data integrity and to also enable hydration for live blogs

### CODE REVIEW

File: src/web/server/document.tsx [hunk 1]
```diff
@@ -88,9 +88,9 @@ export const document = ({ data }: Props): string => {
 	];
 	// We want to only insert script tags for the elements or main media elements on this page view
 	// so we need to check what elements we have and use the mapping to the the chunk name
-	const CAPIElements: CAPIElement[] = CAPI.blocks[0]
-		? CAPI.blocks[0].elements
-		: [];
+	const CAPIElements: CAPIElement[] = CAPI.blocks
+		.map((block) => block.elements)
+		.flat();
```
[the remark this answers is not in the document]
Comment (oliverlloyd) [reply src/web/server/document.tsx:93]: Ah, okay, thanks for pointing that out. I've updated the url to include this feature.

**Comment by oliverlloyd** [comment]
[Flatten elements](https://trello.com/c/Im3W9JNm/2723-flatten-elements)

File: src/web/server/document.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gtrufitt) [reply src/web/server/document.tsx:93]: Don't need it, it's included as a part of es2019 which is a part of the request https://github.com/Financial-Times/polyfill-library/blob/master/polyfills/Array/prototype/flat/config.toml#L2

**Review by gtrufitt — APPROVED** [review]

**Comment by gtrufitt** [comment]
+1 bar removing the un-needed flat from polyfill

**Pull request merged**